[Bug] Fix strong winds calculation

Tempo-anon opened the pull request:
## What are the changes?
Strong winds now reduces rock/electric/ice damage against flying type Pokemon regardless of if the end result is super effective

## Why am I doing these changes?
https://github.com/pagefaultgames/pokerogue/issues/2418
Current strong winds implementation was based off bulbapedia's description and was incorrect. On showdown and in-cart an electric move against mega rayquaza in strong winds should be nve instead of neutral

## What did change?
Removed the `multiplier >= 2` check for strong winds reducing damage by half

### Screenshots/Videos
n/a

## How to test the changes?
Pull

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

CodeTappert reviewed (approved):
As far as i understand this is exactly how it is supposed to work.

If a pokemon has a fyling type (regardless of the second type or if it has one at all), and a attack would be super effective against this flying type then the damage against the flying type is neutral and normaly calculated for a second type.

CodeTappert commented:
@Tempo-anon Please add the message to the game when strong wind triggers.
This is a video of me trying it in ORAS.
https://github.com/pagefaultgames/pokerogue/assets/38758606/f8b70a41-9b35-4953-8e69-f9e3f58befde

Tempo-anon commented:
* Added unit tests
* Added effect trigger message
![](https://cdn.discordapp.com/attachments/1176874654015684739/1253882927951904809/image.png?ex=667778fa&is=6676277a&hm=5f2e424345693d4d8ba1fc02fe611e14b4889c02d1d0491a503b5c3763eda323&)

CodeTappert commented on src/locales/de/weather.ts:
```diff
@@ -40,5 +40,6 @@ export const weather: SimpleTranslationEntries = {
 
   "strongWindsStartMessage": "Alle Flug-Pokémon werden von rätselhaften Luftströmungen geschützt!",
   "strongWindsLapseMessage": "Die rätselhafte Luftströmung hält an.",
+  "strongWindsEffectMessage": "The mysterious air current weakened the attack!",
```
```suggestion
  "strongWindsEffectMessage": "Rätselhafte Luftströmungen haben den Angriff abgeschwächt!",
```

CodeTappert commented on src/locales/fr/weather.ts:
```diff
@@ -40,5 +40,6 @@ export const weather: SimpleTranslationEntries = {
 
   "strongWindsStartMessage": "Un vent mystérieux se lève !",
   "strongWindsLapseMessage": "Le vent mystérieux souffle violemment !",
+  "strongWindsEffectMessage": "The mysterious air current weakened the attack!",
```
```suggestion
  "strongWindsEffectMessage": "Le courant aérien mystérieux affaiblit l’attaque !",
```

CodeTappert commented on src/locales/it/weather.ts:
```diff
@@ -40,5 +40,6 @@ export const weather: SimpleTranslationEntries = {
 
   "strongWindsStartMessage": "È apparsa una corrente d'aria misteriosa!",
   "strongWindsLapseMessage": "La corrente d'aria soffia intensamente.",
+  "strongWindsEffectMessage": "The mysterious air current weakened the attack!",
```
```suggestion
  "strongWindsEffectMessage": "La corrente misteriosa indebolisce l’attacco!",
```

CodeTappert commented on src/locales/ko/weather.ts:
```diff
@@ -41,5 +41,6 @@ export const weather: SimpleTranslationEntries = {
 
   "strongWindsStartMessage": "수수께끼의 난기류가\n비행포켓몬을 지킨다!",
   "strongWindsLapseMessage": "수수께끼의 난기류가 강렬하게 불고 있다",
+  "strongWindsEffectMessage": "The mysterious air current weakened the attack!",
```
```suggestion
  "strongWindsEffectMessage": "수수께끼의 난기류가 공격을 약하게 만들었다!",
```

CodeTappert commented:
Source for all the strings i provided: https://abcboy101.github.io/poke-corpus/#query=The+mysterious+air+current+weakened+the+attack&regex=false&caseInsensitive=true&common=true&script=true&collections=DiamondPearl%2CPlatinum%2CHeartGoldSoulSilver%2CBlackWhite%2CBlack2White2%2CXY%2COmegaRubyAlphaSapphire%2CSunMoon%2CUltraSunUltraMoon%2CLetsGoPikachuLetsGoEevee%2CSwordShield%2CBrilliantDiamondShiningPearl%2CLegendsArceus%2CScarletViolet%2CColosseum%2CXD%2CBattleRevolution%2CBox%2CRanch%2CBank%2CHOME&languages=de-DE%2Cen-US

Tempo-anon commented:
Updated all languages
